### DISCUSSION TITLE
SQLite Read Config Error

### DISCUSSION THREAD

**Discussion by parkershute** [nanomq/nanomq#1449] [Q&A] [answered]
I am constantly seeing this error message in the logs for nanomq:

**ERROR /home/runner/work/nanomq/nanomq/nng/src/supplemental/nanolib/conf_ver2.c:346 conf_basic_parse_ver2: Read config nanomq sqlite failed!**

I have spent a lot of time trying to resolve this error. 

Can I be given some direction of what is causing the read of the config to fail? 

I have even tried running nanomq as root to see if it is a permission issue that will go away, but even as root this error is shown.

Using:
NanoMQ Messaging Engine for Edge Computing & Messaging bus v0.19.5-2

**Comment by parkershute** [comment, marked as answer]
This error appears to be due to port 8083 not open for those who run into this issue.

**Comment by JaylinYu** [comment]
Bingo! You rocks!
cJSON *jso_websocket = hocon_get_obj("listeners.ws", jso);

I will turn this into an issue
